sympy/utilities/tests/test_wester.py

```diff
@@ -971,7 +971,7 @@ def test_M19():
 
 
 def test_M20():
-    assert solveset(sqrt(x**2 + 1) - x + 2, x) == EmptySet()
+    assert solveset(sqrt(x**2 + 1) - x + 2, x) == EmptySet.__class__()
```

Comment:
Why are you checking for the class here?

Reply:
Well the function that is calling needs the instance of the class . So here class of EmptySet is required rather than EmptySet

Reply:
Can you just write `S.EmptySet`? That what we do with something like testing that x is S(1):  `assert x is S.One`.

Reply:
Well I have changed it to`` EmptySet`` because now it is the instance